_pages/en/strategy-target-solution-delivery-model.md

```diff
@@ -831,9 +833,11 @@ A “working IT Product” is a version of the IT Product which allows DevOps te
 A working IT Product must be able to be used by end-users and provide DevOps teams with observation of end-users behaviours.
 
 For the scope of this Strategy, Operating Systems are NOT defined as IT products.
-Therefore should an IT Product depend on an Operating System to run in production, it is compliance with this Guiding Policy.
+Therefore should an IT Product depend on an Operating System to run in production, it is compliance with this Guiding Policy.  
+
+Figure 2. Relationship Model between the different software elements (part 2).  
 
-![How IT Solutions, IT Products, software, and infrastructure relate to each other]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
+![Below is a flowchart with five elements connected to each other.  The flowchart has the following elements displayed in the box: IT Solution, IT Component, Software, Infrastructure. The flow goes from left to right and connects those elements with an arrow. The Configuration element is located below the Software and Infrastructure elements, the latter are pointing to it from both sides.]({{ site.baseurl }}/assets/images/itsolution-itproduct-model.png)
 
```

Comment:
Also I just noticed.....the bottom right box says 'configutation'.  Is that right?  Should it be 'configuration'? If yes we also need to change the image on the ABB page cc @gcharest